pom.xml

```diff
@@ -620,7 +620,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-math3</artifactId>
-                <version>3.2</version>
+                <version>3.5</version>
```

Comment:
I don't insist. I just needed Benchto-driver and Presto agree on the version to get build time check satisfied (and, of course, avoid any problems at run-time). Shall I 3.6.1?